satinfo/sndrD_g15/2020102200

```diff
@@ -0,0 +1,72 @@
+ sndrD1_g15              1  -1    0.900    0.000    2.000   10.000    0.000      1     -1     -1
```

Comment:
Jeff did this sndrd file creep in to tis PR? I am not sure what is the rationale for it. Maybe better remove it from this PR and add it in a separate PR that documents why this has been done.

Reply:
I also see that you added readme file saying there are no dumps after this date. Did you mean for this file to be all "off". As is this file has a bunch of channels turned on.

Reply:
yes, accidemtally branched from a branch with these changes.